server/src/language.js

```diff
@@ -157,16 +156,19 @@ class ProgramCacheItem {
         const program = this.getProgram(request.textDocument.uri);
 
         if (!program) {
-            return {
-                uri: "",
-                range: {
-                    start: { line: -1, character: -1},
-                    end: { line: -1, character: -1}
-                }
-            };
+            return null;
         }
 
         const location = program.getDefinition(request.position);
+
+        if (!location) {
+            return null;
+        }
+
+        if (!location) {
```

Comment:
Oops, I think my merge somehow duplicate this check.

Reply:
Fixed!